### ISSUE TITLE
Unable to locate package kubectx

### ISSUE THREAD

**Issue by guyellis** [ahmetb/kubectx#200]
From CL on Ubuntu 18.04 getting:

```shell
$ sudo apt install kubectx
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package kubectx
```

**Comment by ahmetb** [comment]
I'm not sure if we're in Ubuntu repos. We're in debian repos though.
What do you mean by "CL"? Has this ever worked? Where did you read that it would work on Ubuntu?

**Comment by guyellis** [comment]
CL = Command Line

I followed the instructions on the Readme in this repo: https://github.com/ahmetb/kubectx#debianubuntu

I've never tried this before so don't know if it's ever worked.

I've solved the problem for myself by cloning this repo and "pathing" the scripts so you don't need to solve this for me but you might be interested that the Readme might need updating.

**Comment by ahmetb** [comment]
I think we need to remove the "/Ubuntu" from that title. I'm making the change.

**Comment by ahmetb** [comment]
Fixed in e388bfa.
